src/core/app_backend.tf

```diff
@@ -1012,3 +1012,44 @@ module "app_backend_web_test_api" {
   ]
 
 }
+
+# -----------------------------------------------
+# Alerts
+# -----------------------------------------------
+
+data "azurerm_app_service" "app_backend_app_services" {
+  for_each            = toset(var.app_backend_names)
+  name                = format("%s-app-${each.value}", local.project)
```

Comment:
please use only the interpolation format

Reply:
Right. Done in [fix after review](https://github.com/pagopa/io-infra/pull/296/commits/d04ab9cea130bcf3dcbebbf4d7a819d4ab95b1d1)